### PR TITLE
Refresh landing page

### PR DESCRIPTION
**Pull request by charleswong739**
## Problem

Refresh the landing page design. See [Figma](https://www.figma.com/file/3SMArc1drGfTS25a9D3QCb/See-The-Good?type=design&node-id=0-1&mode=design&t=wwMWz5nTToTOeL5a-0).

## Description of Changes

Edit landing page.
Screenshot 2023-11-26 142707

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by annaa-cho** [comment]
Looks good! Can we add a _little_ bit more of a space between 'good' and the gift emoji, and also move the girl (and the leaves touching her) a little more to the left?

**Comment by nina-huangg** [comment]
Looks so good! Thanks for making your code easy to follow and understand.

File: screens/WelcomeScreen.tsx [hunk 1]
```diff
@@ -5,35 +5,29 @@ import MaskedView from '@react-native-masked-view/masked-view'
...
+                        >
+                            <Text style={styles.titleText}>see the good </Text>
+                        </View>
+                    }
+                >
+                    <LinearGradient colors={['#FF000F', '#A40811']} start={{ x: 0, y: -0.3 }} end={{ x: 0, y: 2 }} locations={[0, 0.7]}>
+                        <Text style={[styles.titleText, { opacity: 0 }]}>see the good </Text>
+                    </LinearGradient>
+                </MaskedView>
+                <Image source={require('../assets/gift.png')} style={styles.gift} />
```
Comment (charleswong739) [inline screens/WelcomeScreen.tsx:29]: Gift image and title text are put into a View horizontally, so it is properly spaced from the title text with a space character at the end of the title text.

File: screens/WelcomeScreen.tsx [hunk 2]
```diff
@@ -155,80 +170,126 @@ const styles = EStyleSheet.create({
...
     titleImage: {
         position: 'absolute',
         bottom: 0,
-        height: '47.51%',
-        width: undefined, // necessary for aspectRatio
-        aspectRatio: 1,
+        paddingRight: '5%',
+        height: '47.31%',
+        width: '100%',
+        left: '-3%',
```
Comment (charleswong739) [inline screens/WelcomeScreen.tsx:188]: Image of girl placed slightly off center

**Comment by annaa-cho** [comment]
Okay perfect love it

**Review by annaa-cho — APPROVED** [review]

**Pull request merged**